delivery_ups_oca/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,16 @@
+To configure this module, you need to:
+
+#. Add a carrier account with delivery type ``ups`` and fill in your credentials (UPS
```

Comment:
Isn't 'delivery type' instead of 'carrier account' the model?

Reply:
This describes the configuration in the UI where it is actually called Shipping Method and Delivery Method (inconsistently) with Provider "UPS". I changed it accordingly.